### ISSUE TITLE
Reader comments render issue.

### ISSUE THREAD

**Issue by Nness** [alistapart/AListApart#438]
Example page: https://alistapart.com/article/dropdowns/

Microsoft Edge 80.0.361.66 (Chromium)
![image](https://user-images.githubusercontent.com/1712143/76558233-cc965b80-6501-11ea-814c-2c4ec084bc7a.png)

Firefox 73.01
![image](https://user-images.githubusercontent.com/1712143/76558305-efc10b00-6501-11ea-8fa7-e1f3a54b5ac6.png)

![image](https://user-images.githubusercontent.com/1712143/76558430-30208900-6502-11ea-9065-566f41c6e6ca.png)

The width of comment is caused by list's width style
![image](https://user-images.githubusercontent.com/1712143/76559333-d4ef9600-6503-11ea-979f-baffd1660c1b.png)

The fix is to add `width: unset` to the comment class or remove `width: 160px` in the first place.

Sample of fixed comment
![image](https://user-images.githubusercontent.com/1712143/76559412-fea8bd00-6503-11ea-8a51-e6fd8d239120.png)

Of course there are other issues with comment content itself.

**Comment by dpanov** [comment]
Good catch. :-) It's not just the comments, there's a `<style>` element with some CSS in it that messes up with the whole page. Here's the header, for example:

![a-list-apart](https://user-images.githubusercontent.com/1931510/77252159-19450980-6c5b-11ea-8d6d-6d3df4cece55.png)

Unfortunately, the A List Apart team isn't really paying much attention to this repository and the issues posted here, so I don't think this will be fixed any time soon.
